src/TodoList.js

```diff
@@ -0,0 +1,111 @@
+import Clock from "./Clock";
+import TodoInput from "./TodoInput";
+import ListContainer from "./ListContainer";
+import { useEffect, useState } from "react";
+import { styled } from "styled-components";
+
+const TodoList = () => {
+  const [data, setData] = useState([]);
+
+  //콘솔창에서 data 확인
+  useEffect(() => {
+    console.log(data);
+  });
+
+  //localstorage 받아오기
+  useEffect(() => {
+    const localData = localStorage.getItem("todos");
+    if (localData) {
+      const parsedDataList = JSON.parse(localData);
+      setData(parsedDataList);
+    }
```

Comment:
이 부분은 이렇게 간단화할 수 있을 것 같습니다!

```suggestion
    setData(JSON.parse(localStorage.getItem('todos')||'[]'));
```